<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>State</string>
    </attribute>
    <widget class="QLabel" name="contractTxIdValueLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;txid&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="StateTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ContractTxIdTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Contract TxId:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateValueLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;buying mode state&gt;</string>
     </property>
    </widget>
    <widget class="QTreeView" name="sellersTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>771</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Terms</string>
    </attribute>
    <widget class="QLabel" name="MaxPriceTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum price/piece (sats):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxRefundDelayTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum refund delay (s):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxContractFeePrKbTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum contract fee (sats/Kb):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxPriceValueLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;price&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxContractFeePrKbValueLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;fee&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="minNumberOfSellersTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum number of sellers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxRefundDelayValueLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;delay&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="minNumberOfSellersValueLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;#sellers&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateTermsPushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Policy</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
